commons-packet/commons-packet-service/configure_start.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+echo "Starting prerequisite setup..."
+
+echo "Downloading mandatory dependencies..."
+
+wget -q --show-progress "${kernel_ref_idobjectvalidator_url}" \-O "${loader_path_env}/kernel-ref-idobjectvalidator.jar"
```

Shellcheck (0.11.0)

[warning] kernel_ref_idobjectvalidator_url is referenced but not assigned.

(SC2154)